### ISSUE TITLE
Error when setting train schedule to invalid rail target locks up debugger

### ISSUE THREAD

**Issue by muppet9010** [justarandomgeek/vscode-factoriomod-debug#31]
If i set a train schedule record of type rail to a non existent entity the debugger just stops. It doesn't freeze, but nothing progresses and no error is shown. The base game run without debugger shows an invalid rail entity error. 

Concept code that breaks when in a mod, just needs a train somewhere and NO track at {0,0}:
```
local train = GET A LUATRAIN REFERENCE
train.schedule = {
    current = 1,
    records = {
        {
            rail = game.surfaces[1].find_entity("straight-rail", {0,0})
        }
    }
}
```

**Comment by muppet9010** [comment]
seems more wide spread as if I pass in invalid arguments to LuaEntity.mine() the debugger also stops progressing at that point with no message shown. If I run the same mod code in natural Factorio 1.1.6 it gives an error.
https://lua-api.factorio.com/latest/LuaEntity.html#LuaEntity.mine
Called the below, the second argument is invalid as misread the API doc: 
`ENTITY.mine(anInventory, game.forces[1], true, true)`

**Comment by justarandomgeek** [comment]
This should be resolved in 1.1.3
